### PR TITLE
changelog: Update for 2.2.

### PR DESCRIPTION
**Pull request by HansKristian-Work**
Signed-off-by: Hans-Kristian Arntzen <post@arntzen-software.no>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,47 @@
...
+
+### Regression fixes
+
+- Fix regression in Horizon Zero Dawn for screen space reflections on water surfaces.
+
+### Stability fixes
+
+- Greatly improve stability on Polaris or older cards for certain titles.
+  Crashes which used to happen in Horizon Zero Dawn and Death Stranding seem to have disappeared
+  after the memory allocation rewrite.
```
Comment (nutta-git) [inline CHANGELOG.md:30]: Revised; 

`Crashes in Horizon Zero Dawn and Death Stranding fixed with memory allocation rewrite.`

fixed can be interchanged with; resolved or improved

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,47 @@
...
 
+## 2.2
+
+This release is mostly a maintenance release which fixes bugs and regressions.
+It also unblocks significant future feature development.
+
+### Workaround removals
+
+- Remove old `force_bindless_texel_buffer` workaround with
+  a more correct and performant workaround.
```
Comment (doitsujin) [inline CHANGELOG.md:11]: s/Remove/Replace
s/workaround/implementation since it's always on now

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,47 @@
...
+  Death Stranding and Cyberpunk 2077 (and probably other games as well) do the right thing by default without the hack now.
+- Remove old workaround for occlusion queries which was enabled for AC: Valhalla.
+  Improves performance in AC: Valhalla.
```
Comment (doitsujin) [inline CHANGELOG.md:14]: Does it? I don't think Valhalla uses queries enough for this to make a difference.
I'd just bunch this up with the texel buffer workaround (name was `disable_query_optimization`).

Comment (HansKristian-Work) [reply CHANGELOG.md]: Fixed.

File: CHANGELOG.md (hunk 2 above)
Comment (HansKristian-Work) [reply CHANGELOG.md]: Fixed.

**Review by doitsujin — APPROVED** [review]

**Pull request merged**